scripts/commitlint/index.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('@octokit/rest')
+const lint = require('@commitlint/lint')
+
+const { rules } = require('../../commitlint.config')
+
+function getPrNumber() {
+  const prUrl = process.argv[2]
+  const prUrlParts = prUrl.split('/')
+
+  return prUrlParts[prUrlParts.length - 1]
+}
+
+function getCommitsByPr(number) {
+  const github = new Octokit()
+  return github.pulls.listCommits({
+    owner: 'Royal-Navy',
+    repo: 'standards-toolkit',
+    pull_number: number,
+  })
+}
+
+function lintCommits(commits) {
+  const promises = commits.data.map(commit => {
```

Comment:
```suggestion
  const promises = commits.data.map(({ commit: { commit: message }}) => {
```

Can take it or leave it with this kind of thing - arguably not better, just different.